packages/stan/README.md

```diff
@@ -0,0 +1,203 @@
+// This property (laziness) is important for performance reasons.
+sumInstance.subscribe( () => {
+    console.log( registry.get( sum ) );
+} );
+
+// This edits counter1, triggering a resolution of sumInstance which triggers the console.log above.
+registry.set( counter1, 2 ); // now both counters equal 2 which means sum will print 4.
+registry.set( counter1, 4 ); // prints 6
+```
+
+### Async derived atoms
+
+Derived atoms can use async functions to compute their values. They can for instance trigger some REST API call and returns a promise.
+
+```js
+const sum2 = createDerivedAtom(
+    async ( { get } ) => {
+        const val1 = await Promise.resolve(10);
+        return val1 * get( counter );
+    } 
+);
+```
+
+The value of async atoms will be equal to `null` until the resolution function finishes.
+
+### Bindings
+
+It is important to note that stan instance and registries API are low-level APIs meant to be used by developpers to build bindings for their preferred frameworks. By in general, a higher-level API is preferred.
+
+Currently available bindings:
+
+-   `@wordpress/data`: WordPress data users can continue to use their existing high-level APIs useSelect/useDispatch (selectors and actions) to access the atoms. The selectors are just high-level atoms that can rely on lower-level ones and the actions are just functions that trigger atom setters. The API for `@wordpress/data` store authors to bridge the gap is still experimental.
+
+## API Reference
+
+<!-- START TOKEN(Autogenerated API docs) -->
+
+<a name="createAtom" href="#createAtom">#</a> **createAtom**
+
+Creates a basic atom.
+
+_Parameters_
+
+-   _initialValue_ `T`: Initial value in the atom. \*
+-   _config_ `[WPCommonAtomConfig]`: Common Atom config.
+
+_Returns_
+
+-   `WPAtom<T>`: Createtd atom.
+
+<a name="createAtomFamily" href="#createAtomFamily">#</a> **createAtomFamily**
+
+_Parameters_
+
+-   _resolver_ `WPAtomFamilyResolver<T>`: Atom resolver.
+-   _updater_ `WPAtomFamilyUpdater<T>`: Atom updater.
+-   _atomConfig_ `[WPCommonAtomConfig]`: Common Atom config.
+
+_Returns_
+
+-   (unknown type): Atom Family Item creator.
+
+<a name="createAtomRegistry" href="#createAtomRegistry">#</a> **createAtomRegistry**
+
+Creates a new Atom Registry.
+
+_Parameters_
+
+-   _onAdd_ `RegistryListener`: 
+-   _onDelete_ `RegistryListener`: 
+
+_Returns_
+
+-   `WPAtomRegistry`: Atom Registry.
+
+<a name="createDerivedAtom" href="#createDerivedAtom">#</a> **createDerivedAtom**
+
+Creates a derived atom.
+
+_Parameters_
+
+-   _resolver_ `WPDerivedAtomResolver<T>`: Atom Resolver.
+-   _updater_ `WPDerivedAtomUpdater<T>`: Atom updater.
+-   _config_ `[WPCommonAtomConfig]`: Common Atom config.
+
+_Returns_
+
+-   `WPAtom<T>`: Createtd atom.
+
+<a name="createStoreAtom" href="#createStoreAtom">#</a> **createStoreAtom**
+
+Creates a store atom.
+
+_Parameters_
+
+-   _subscribe_ (unknown type): Subscribe to state changes.
```

Comment:
Types for params are missing.

Reply:
These are inline typed which I think is better for these, I don't want to create dedicated types for these basic arguments. I feel it's more a tool issue.

Reply:
Huh, yes, let's leave it as is. We definitely need to improve how this tool works as it clearly works against us when using TypeScript syntax.